code/modules/food_and_drinks/kitchen_machinery/oven_new.dm

```diff
@@ -43,3 +43,27 @@
 	for(var/obj/item/weapon/stock_parts/micro_laser/M in component_parts)
 		E += M.rating
 	efficiency = round((E/2), 1) // There's 2 lasers, so halve the effect on the efficiency to keep it balanced
+
+/obj/machinery/kitchen_machine/oven/special_attack(obj/item/weapon/grab/G, mob/user)
+	if(ishuman(G.affecting))
+		if(G.state < GRAB_AGGRESSIVE)
+			to_chat(user, "<span class='warning'>You need a better grip to do that!</span>")
+			return 1
+		var/mob/living/carbon/human/C = G.affecting
+		var/obj/item/organ/external/head/head = C.get_organ("head")
+		if(!head)
+			to_chat(user, "<span class='warning'>This person doesn't have a head!</span>")
+			return 1
+		C.visible_message("<span class='danger'>[user] bashes [C]'s head in [src]'s door!</span>", \
+						"<span class='userdanger'>[user] bashes your head in [src]'s door! It feels rather hot in the oven!</span>")
+		C.emote("scream")
+		user.changeNext_move(CLICK_CD_MELEE)
+		if(!(head.status & ORGAN_ROBOT))
```

Comment:
They will take increased brute damage. I feel this just balances out the damage itself a little more for this particular case.